builder/builder-next/builder.go

```diff
@@ -209,6 +209,12 @@ func (b *Builder) Build(ctx context.Context, opt backend.BuildConfig) (*builder.
 		frontendAttrs["no-cache"] = ""
 	}
 
+	if opt.Options.PullParent {
+		frontendAttrs["image-resolve-mode"] = "pull"
```

Comment:
Thinking if it would be good to use the values that are / will be defined in `source.ResolveMode`

Reply:
@thaJeztah actually, could do it in a follow up once buildkit has a String() method on it. I don't want to rely on llb package at this level of abstraction.

Reply:
https://github.com/moby/buildkit/pull/577

Reply:
Yup; definitely not a blocker (same for my other comment) 👍